vite.config.js

```diff
@@ -1,7 +1,20 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
 
 // https://vite.dev/config/
+
+//make sure to install Vitest
+//npm install --save-dev vitest @testing-library/react @testing-library/jest-dom
+
 export default defineConfig({
   plugins: [react()],
-})
+  test: {
+    // allow `describe`, `it`, `expect` globally
+    globals: true,
```

Comment:
I didn't get it at first. Okay, I get it now. `describe`, `it` and `expect` are global so we don't  have to import tham.